src/libYARP_dev/src/devices/ControlBoardWrapper/SubDevice.cpp

```diff
@@ -274,7 +274,7 @@ bool SubDevice::attach(yarp::dev::PolyDriver *d, const std::string &k)
     }
 
     int subdevAxes;
-    if(!pos->getAxes(&subdevAxes))
+    if(!pos && !pos->getAxes(&subdevAxes))
```

Comment:
`!pos` and `!pos->getAxes(&subdevAxes)` are both error conditions, so I think this should be an or (`||`), not and (`&&`).

Reply:
Right, fixed